packages/esm-patient-search-app/src/patient-search-page/refine-search/person-attribute-field.component.tsx

```diff
@@ -0,0 +1,232 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react';
+import { ComboBox, InlineLoading, InlineNotification, TextInput, TextInputSkeleton } from '@carbon/react';
+import { useTranslation } from 'react-i18next';
+import { useAttributeConceptAnswers, useLocations, usePersonAttributeType } from './person-attributes.resource';
+import { type AdvancedPatientSearchState, type SearchFieldConfig } from '../../types';
+import styles from './search-field.scss';
+
+export interface PersonAttributeFieldProps {
+  field: SearchFieldConfig;
+  formState: AdvancedPatientSearchState;
+  inTabletOrOverlay: boolean;
+  isTablet: boolean;
+  onInputChange: (fieldName: string) => (evt: { target: { value: string } } | { name: string }) => void;
```

Comment:
This is a lot of prop passing from RefineSearch to the end fields.

Can we do the whole form handling with RHF, rather than handling the form input manually. I know I implemented it the manual way, since we didn't use the RHF at that time. We can take this in the next iteration.